### DISCUSSION TITLE
Custom Lists

### DISCUSSION THREAD

**Discussion by LAwmand** [magicbug/Cloudlog#1830] [Ideas / Feature Requests]
Hi,
Excuse me if this is a feature already and I haven't seen how to do it. (I've got form for this).

I would like to see a "list version" of custom maps. 

So, for example if I wanted a record of my 23cm qso's on CW in the last 6 months I could get a list up which I could then see what was what, and if possible, export.

73 David G0LBK

**Comment by dg9vh** [comment]
This could be done with the advanced search functionality. Just click on
search and then advanced search.

Here you can also save your query for later reusing.

73 Kim

Am Mi., 7. Dez. 2022 um 12:46 Uhr schrieb LAwmand ***@***.***
>:

> Hi,
> Excuse me if this is a feature already and I haven't seen how to do it.
> (I've got form for this).
>
> I would like to see a "list version" of custom maps.
>
> So, for example if I wanted a record of my 23cm qso's on CW in the last 6
> months I could get a list up which I could then see what was what, and if
> possible, export.
>
> 73 David G0LBK
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1830>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OW47FPTDEUVL2DUK7DWMB2H7ANCNFSM6AAAAAASWWR7XE>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
DARC e.V., Ortsverband Q03, Neunkirchen

**Comment by LAwmand** [comment]
Thank you Kim.

**Comment by LAwmand** [comment]
I've just looked at advanced  searches, its too advanced for me to do what I want.
All I was looking at all contacts on a chosen band between 2 dates, although I get the gist, its not easy.
I don't think i'll bother, its probably easier to export the log and do it in a spreadsheet or something.

Please consider a list version of custom maps..

**Comment by AndreasK79** [comment]
I see your point. The advanced search doesn't have greater or less in filter.

If I understand you correctly, you would like a logbook view with filtering?

**Reply by LAwmand** [reply]
Yes, simply put, that's it. A logbook with (Basic) filtering.
For me personally date, band, mode, Propagation would do everything I need,
Regards
David

**Reply by AndreasK79** [reply]
Maybe something like this?
![image](https://user-images.githubusercontent.com/6977712/208265704-848001fb-5af3-49b1-b92c-d583771aad99.png)
Heavily inspired by @hugosilvaparagramanet's QSL Manager. I took his PR and customized it.

**Reply by magicbug** [reply]
That would be pretty sweet

**Reply by AndreasK79** [reply]
The functionality is 99% there and working, but the table is not updating dynamically.

@magicbug I've called it logbookadvanced right now. Not sure where to put it. In my dev I've just replaced the logbook-link.

**Reply by magicbug** [reply]
Hmm, yes does seem to be a replacement for it

**Reply by AndreasK79** [reply]
I guess we'll find out when it's closer to done. This is just a teaser :)

**Comment by LAwmand** [comment]
Yes that is more than enough for what I would need.
The ability to export it would be good, but it's fine like that.
Good work
David

**Reply by AndreasK79** [reply]
Take a look at the picture. You have a button called Create ADIF.

**Comment by LAwmand** [comment]
Excellent, I'm viewing on a mobile.
Sorry I missed that.

**Reply by AndreasK79** [reply]
No worries :)

**Comment by AndreasK79** [comment]
#1847 now added. Feature complete, except for storing the returned data from callbook lookup.

**Reply by LAwmand** [reply]
Thank you Andreas, I'm happy you considered it worthwhile and made it happen. You are a star!
Best wishes
David G0LBK

**Reply by AndreasK79** [reply]
Happy to help David. It was several factors that lead to it, but you pointed out some shortcomings in the advanced search. It is probably too advanced for those now knowing SQL, and I couldn't find less or greater than either.

Besides an uncomplete feature was submitted to Cloudlog several months ago, which I modified and expanded upon.

**Reply by AndreasK79** [reply]
It's merged in to the dev branch, so if you use that, you can try it out. If not, need to wait for a new release.
